src/ruvsarpur.py

```diff
@@ -332,6 +332,12 @@ def runMain():
       local_filename = "{0} - {1} ({2}).mp4".format(item['title'], 
                                                    item['showtime'][:10],
                                                    item['pid'])
+
+    # if on Windows, remove symbols that the filesystem doesn't like 
+    if(os.name == "nt"):      
+      for ch in ["\\","/",":","?","*","\"","<",">","|"]:
+         if ch in local_filename:
```

Comment:
Good start, missing tab, newline and null char. It would probably also be slightly better to use a single statement here rather than iterate over every character, e.g. 
import re
local_filename = re.sub(r"[\"//:\\<>|\?\*\n\r\t\0]", "", local_filename)